ECHO-COVID19.ipynb

```diff
@@ -228,158 +206,11 @@
    },
    "outputs": [],
    "source": [
-    "charter(stack_data, 'ACTUAL_END_DATE', \"%m/%d/%Y\", mnth, mnth_name, 'STATE_EPA_FLAG', \n",
+    "charter(stack_data, 'ACTUAL_END_DATE', \"%m/%d/%Y\", mnth_name, 'STATE_EPA_FLAG', \n",
     "        \"Number of stack tests\", \"air_monitoring.csv\", \"# of pollutants stack-tested\", \n",
```

Comment:
This needs to be clarified: is it number of tests, or number of pollutants? The markdown, chart title, and chart legend don't seem to be in agreement

Reply:
The figure legend also needs to be modified now that it's 2000-2020 (is this part of the charter utiity?) should be "across 2000-2020" instead of "across ECHO history"

Reply:
fixed